### PR TITLE
Adds pre-warning to immovable rod event

### PR DESCRIPTION
**Pull request by Goldsteel**
Adds a pre-warning message to the immovable rod random event, alerting the crew a few seconds before disaster. The message has also changed.

:cl:
 * rscadd: Adds pre-warning to immovable rod random events with a new message reflecting the nature of the event.

### CODE REVIEW

File: code/datums/helper_datums/command_alerts.dm [hunk 1]
```diff
@@ -464,8 +464,8 @@ The access requirements on the Asteroid Shuttles' consoles have now been revoked
 
 /datum/command_alert/immovable_rod
 	name = "Immovable Rod (\"What The Fuck Was That?\")"
-	alert_title = "General Alert"
-	message = "What the fuck was that?!"
+	alert_title = "Local Bluespace Sensor Report"
+	message = "Alert: An anomalous object with extremely high relative velocity is on a collision course with this station. Heavy structural damage may result."
```
Comment (Intigracy) [inline code/datums/helper_datums/command_alerts.dm]:
```suggestion
	message = "Alert: The station appears to be on a collision course with an anomalous object perfectly suspended in space. Heavy structural damage may result."
```

Comment (Intigracy) [reply code/datums/helper_datums/command_alerts.dm]: Rods don't have velocity, they're not moving.

Comment (Goldsteel) [reply code/datums/helper_datums/command_alerts.dm]: I put relative velocity but sure, that works too.

**Comment by incakola** [comment]
I kind of like that the rod is unexpected.
Just a breach and on occasion a gib.
Like a bolt from Zeus.
Unless you are stating in the warning the direction the rod travels then it's not very useful no?

**Comment by Intigracy** [comment]
What the fuck is that?!
spawn(100)
What the fuck was that?!

😏

**Pull request merged**